BotghaniNotepad.js

```diff
@@ -1,21 +1,16 @@
 import React, { PureComponent } from 'react'
 import { Provider } from 'react-redux'
 import { createStackNavigator, createDrawerNavigator, createAppContainer } from 'react-navigation'
-import { PERSIST_STATE } from 'react-native-dotenv'
 import { PersistGate } from 'redux-persist/integration/react'
+import store, { persister } from './src/state/store'
+import env from './env'
```

Comment:
you COULD just destrcuture persist state, but it's not a necessary request. just a suggestion :)